specification/cognitiveservices/Vision.ImageAnalysis/routes.tsp

```diff
@@ -34,8 +34,8 @@ op analyzeStream is Azure.Core.RpcOperation<
     @query("overload")
     overload: "stream";
 
-    @doc("Impl Detail")
-    @header("Content-Type")
+    @doc("The format of the image stream")
```

Comment:
Not so much the format of the image, but the HTTP request payload... Maybe just say "The content type of the HTTP request payload"